### PR TITLE
feat: add simple wrapper to test workflows

### PR DESCRIPTION
**Pull request by Niraj-Kamdar**
- Relates: #983
- Closes: #997

### CODE REVIEW

File: packages/js/client/src/__tests__/e2e/workflow-test-cases.ts [hunk 1]
```diff
@@ -1,81 +1,189 @@
...
-  jobs: {
-    cases: {
-      steps: [
-        {
-          uri: "ens/testnet/simple-storage.eth",
-          method: "deployContract",
-          args: {
-            connection: null,
-          },
+export interface WorkflowTestCase {
```
Comment (nerfZael) [inline packages/js/client/src/__tests__/e2e/workflow-test-cases.ts:5]: Why do we have test cases as a separate file if we're only using them in one test file?

File: packages/js/client/src/__tests__/e2e/workflow.spec.ts [hunk 2]
```diff
@@ -1,75 +1,39 @@
...
+  let client: PolywrapClient;
+
   beforeAll(async () => {
     await initTestEnvironment();
+
+    await buildWrapper(
+      path.join(GetPathToTestWrappers(), "wasm-as", "simple-calculator")
+    );
+
+    client = await getClientWithEnsAndIpfs();
```
Comment (nerfZael) [inline packages/js/client/src/__tests__/e2e/workflow.spec.ts]: You can just use getClient

File: packages/js/client/src/__tests__/e2e/workflow.spec.ts [hunk 3]
```diff
@@ -1,75 +1,39 @@
...
+import { testCases } from "./workflow-test-cases";
+import path from "path";
 
 jest.setTimeout(200000);
 
 describe("workflow", () => {
+  let client: PolywrapClient;
+
   beforeAll(async () => {
     await initTestEnvironment();
```
Comment (nerfZael) [inline packages/js/client/src/__tests__/e2e/workflow.spec.ts]: No need for test-env, that was the point of all this :D

**Review by nerfZael — CHANGES_REQUESTED** [review]

File: packages/js/client/src/__tests__/e2e/workflow-test-cases.ts (hunk 1 above)
Comment (Niraj-Kamdar) [reply packages/js/client/src/__tests__/e2e/workflow-test-cases.ts:5]: Just for more readability

File: packages/test-cases/cases/wrappers/wasm-as/simple-calculator/package.json [hunk 4]
```diff
@@ -0,0 +1,8 @@
+{
+  "name": "test-case-simple-calculator",
+  "private": true,
+  "dependencies": {
+    "@polywrap/wasm-as": "0.0.1-prealpha.91",
```
Comment (cbrzn) [inline packages/test-cases/cases/wrappers/wasm-as/simple-calculator/package.json:5]:
```suggestion
    "@polywrap/wasm-as": "0.1.0",
```

**Review by cbrzn — APPROVED** [review]
This is an amazing example that I will use to implement workflows in the integrations repository. Thanks for this :)

**Comment by Niraj-Kamdar** [comment]
@cbrzn You mean demo right?

**Pull request merged**